packages/inference/src/tasks/audio/automaticSpeechRecognition.ts

```diff
@@ -28,8 +28,6 @@ export async function automaticSpeechRecognition(
 	return res;
```

Comment:
Should use `providerHelper.getResponse(res)` here? (for e.g. `FalAIAutomaticSpeechRecognitionTask`)